src/components/UI/Menu/MenuItem/MenuItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { MenuItem as MenuItemElement } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import styles from './MenuItem.module.css';
+
+export interface MenuItemProps {
+  title: string;
+  path: string;
+  onClickHandler: any;
+  className?: string;
+}
+
+const MenuItem: React.SFC<MenuItemProps> = (props) => {
+  let menuItemClass = '';
+  if (props.className === 'Warning') {
```

Comment:
it's a minor thing but do you think we should rename to "danger", like other UI frameworks does?
btn-danger (red)
btn-warning (yellow)
btn-success (green)

Reply:
Since we are using objects, I have changed this to "Danger".